nsv13/code/modules/munitions/ammunition/broadside_ammo.dm

```diff
@@ -0,0 +1,132 @@
+/obj/item/ship_weapon/ammunition/broadside_shell
+	name = "\improper SNBC Type 1 Shell"
+	desc = "A large packed shell, complete with powder and projectile, ready to be loaded and fired."
+	icon_state = "broadside"
+	lefthand_file = 'nsv13/icons/mob/inhands/weapons/bombs_lefthand.dmi'
+	righthand_file = 'nsv13/icons/mob/inhands/weapons/bombs_righthand.dmi'
+	icon = 'nsv13/icons/obj/munitions.dmi'
+	w_class = WEIGHT_CLASS_BULKY
+	projectile_type = /obj/item/projectile/bullet/broadside
+
+/obj/item/ship_weapon/ammunition/broadside_shell/plasma
+	name = "\improper SNBC Type P Shell"
+	desc = "A large packed shell, complete with plasma and projectile, ready to be loaded and fired."
+	icon_state = "broadside_plasma"
+	lefthand_file = 'nsv13/icons/mob/inhands/weapons/bombs_lefthand.dmi'
+	righthand_file = 'nsv13/icons/mob/inhands/weapons/bombs_righthand.dmi'
+	icon = 'nsv13/icons/obj/munitions.dmi'
+	w_class = WEIGHT_CLASS_BULKY
+	projectile_type = /obj/item/projectile/bullet/broadside/plasma
+
+/obj/item/ship_weapon/ammunition/broadside_shell/can_be_pulled()
+	return TRUE
+
+/obj/machinery/broadside_shell_packer
+	name = "\improper Broadside Shell Packer Bench"
+	desc = "An automated table that packs broadside shell casings, just add components!"
+	icon = 'nsv13/icons/obj/munitions/packing_bench.dmi'
+	icon_state = "packing_bench"
+	density = TRUE
+	anchored = TRUE
+	circuit = /obj/item/circuitboard/machine/broadside_shell_packer
+	bound_width = 64
+	var/casing_amount = 0
+	var/load_amount = 0
+	var/bag_amount = 0
+	var/amount_to_pack = 5
+	var/obj/item/powder_bag/plasma/plasma = FALSE
+	var/obj/item/powder_bag/gunpowder = FALSE
+	var/static/list/thingy = typecacheof(list(
+		/obj/item/ship_weapon/parts/broadside_casing,
+		/obj/item/ship_weapon/parts/broadside_load,
+		/obj/item/powder_bag))
+
+/obj/machinery/broadside_shell_packer/examine(mob/user)
+	. = ..()
+	. += "It has [casing_amount] casings, [load_amount] loads, and [bag_amount] bags."
+	. += "The Packer requires 5 Casings, 5 Loads and 1 Powder Bag to pack 5 Broadside Shells."
+
+/obj/machinery/broadside_shell_packer/attackby(obj/item/I, mob/living/user, params)
+	. = ..()
+	if(thingy[I.type])
+		if(istype(I, /obj/item/ship_weapon/parts/broadside_casing))
```

Comment:
You might want to consider adding return TRUE to the end of each of these istypes and have the parent call at the end, since otherwise you smack the table with the casing/load.

Reply:
Tried, couldn't get it to play nice. It just messed with the sprite overlays and such.